pyproject.toml

```diff
@@ -148,3 +149,18 @@ force_sort_within_sections = true
 
 [tool.pytest.ini_options]
 addopts = "--disable-warnings"
+
+[tool.coverage.run]
+omit = [
+    "*/local*",
+    "*/rule*",
+    "deprecated_functions.py",
+    "*/nnpdf_data/commondata/*",
+    "*/nnpdf_data/filter_utils/*",
+]
+[tool.coverage.paths]
+cipaths = [ # In order to combine files from both the ubuntu and macos runners
+    "./", # CI working directory
+    "/Users/runner/work/nnpdf/nnpdf",
+    "/home/runner/work/nnpdf/nnpdf"
+]
```

Comment:
I think that tests folder could be omitted.
Coverage also counts them in the statitistics of how much code is covered

Reply:
Let's see how it changes. I guess the tests are around 100% by definition unless we have pieces that are not used?

Reply:
It gets down to 65%... no longer nice.